stripe/src/main/java/com/stripe/android/payments/StripeBrowserLauncherViewModel.kt

```diff
@@ -17,4 +28,41 @@ internal class StripeBrowserLauncherViewModel : ViewModel() {
             ).toBundle()
         )
     }
+
+    fun fireAnalytics(
```

Comment:
actually I think the convention is logXYZ just comparing to PaymentAutheWebViewActivityViewModel, and fireAnalytics is private to the viewModel